cli/cmd/runserver.go

```diff
@@ -113,24 +122,47 @@ func runserverFunc(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
+	cfg, err := clientcmd.BuildConfigFromFlags(kubeServer, kubeConfig)
+	if err != nil {
+		fmt.Fprintf(os.Stderr,"Failed to build kubeconfig: %v\n", err)
+		os.Exit(1)
+	}
+	karydiaClientset, err := clientset.NewForConfig(cfg)
+	if err != nil {
+		fmt.Fprintf(os.Stderr,"Failed to build karydia clientset: %v\n", err)
+		os.Exit(1)
+	}
+
+	karydiaConfig, err := karydiaClientset.KarydiaV1alpha1().KarydiaConfigs().Get(viper.GetString("config-custom-resource"), metav1.GetOptions{})
+	if err != nil {
+		fmt.Fprintf(os.Stderr,"Failed to load karydia config: %v\n", err)
+		os.Exit(1)
+	}
+	fmt.Fprintf(os.Stderr, "KarydiaConfig Name: %s\n", karydiaConfig.Name)
```

Comment:
Two questions:
- Why print to `stderr` here? Isn't this expected behavior?
- At least in other karydia packages, a logger is used instead of directly printing to any output.

Reply:
The point from your second question is already mentioned at #89 and should be handled there.